Makefile.am

```diff
@@ -352,6 +353,7 @@ libarchive_test_SOURCES= \
 	libarchive/test/test_archive_write_add_filter_by_name.c \
 	libarchive/test/test_archive_write_set_filter_option.c \
 	libarchive/test/test_archive_write_set_format_by_name.c \
+    libarchive/test/test_archive_write_set_format_filter_by_ext.c \	
```

Comment:
Please use the same indentation, please alphabetize.  (It looks like a few of these filenames are out of order.)